internal/pkg/argocd/argocd.go

```diff
@@ -126,7 +126,11 @@ func generateArgocdAppDiff(ctx context.Context, app *argoappv1.Application, proj
 				foundDiffs = true
 			}
 
-			diffElement.Diff, err = diffLiveVsTargetObject(live, target)
+			if keepDiffData {
+				diffElement.Diff, err = diffLiveVsTargetObject(live, target)
+			} else {
+				diffElement.Diff = "✂️ ✂️  Redacted ✂️ ✂️ \nUnset component-level configuration key `disableArgoCDDiff` to see diff content."
+			}
 			if err != nil {
 				return false, nil, fmt.Errorf("Failed to diff live objects: %w", err)
 			}
```

Comment:
not related to this pr but i think we already checked the error here -> https://github.com/commercetools/telefonistka/blob/e35f54e3bec2444c5a3433a650e47d3af886144b/internal/pkg/argocd/argocd.go#L102-L104